tools/src/main/scala/scala/scalanative/build/Discover.scala

```diff
@@ -2,7 +2,7 @@ package scala.scalanative
 package build
 
 import java.nio.file.{Files, Path, Paths}
-import scala.collection.JavaConverters._
+import scala.jdk.CollectionConverters._
```

Comment:
This seems to be used only at
https://github.com/scala-native/scala-native/blob/7855ba939d7ce16b7026a99c9fc978ab5fa04bb0/tools/src/main/scala/scala/scalanative/build/Discover.scala#L86-L93
Perhaps we could rewrite it as follows to get rid of the collection converters:
```scala
      val linesIter = Files.readAllLines(targetll).iterator()
      while (linesIter.hasNext()) {
        val line = linesIter.next()
        if (line.startsWith("target triple"))
          return line.split("\"").apply(1)
      }
      fail
```